    Removed file as a prerequisite for bug 1213267 - [7.3 FEAT] Configure
    glibc with --with-cpu=power8 --with-tune=power8 for RHEL7.3 ppc64le.

Part of this upstream commit:

commit 5e6a4d4b9eb579c5ca606c0eed1f2f40e405a5f1
Author: Adhemerval Zanella <azanella@linux.vnet.ibm.com>
Date:   Fri Dec 13 14:29:27 2013 -0500

    PowerPC: Adjust multiarch Implies for PowerPC64

diff --git a/sysdeps/unix/sysv/linux/powerpc/powerpc64/power8/Implies b/sysdeps/unix/sysv/linux/powerpc/powerpc64/power8/Implies
deleted file mode 100644
index fad2505..0000000
--- a/sysdeps/unix/sysv/linux/powerpc/powerpc64/power8/Implies
+++ /dev/null
@@ -1,2 +0,0 @@
-powerpc/powerpc64/power8/fpu
-powerpc/powerpc64/power8
